tfs/src/main/java/hudson/plugins/tfs/TeamPushTrigger.java

```diff
@@ -119,13 +120,13 @@ public void run() {
             boolean shouldSchedule = bypassPolling;
             String changesDetected = "";
             if (!bypassPolling) {
-                if (runPolling()) {
+                if (runPolling() || StringUtils.isNotBlank(gitCodePushedEventArgs.commit)) {

```

Comment:
Can you add a comment here to indicate why the gitCodePushedEventArgs is needed?